editor/app/src/AnalyticsPlayModeSetup.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Firebase.Editor {
+
+[InitializeOnLoad]
+internal static class AnalyticsPlayModeSetup {
+    private const string DestinationDir = "./";
+
+    static AnalyticsPlayModeSetup() {
+        EditorApplication.playModeStateChanged += OnPlayModeStateChanged;
+    }
+
+    private static void OnPlayModeStateChanged(PlayModeStateChange state) {
+        if (state == PlayModeStateChange.EnteredPlayMode) {
+            if (Application.platform == RuntimePlatform.WindowsEditor) {
+                Type firebaseAnalyticsType = Type.GetType("Firebase.Analytics.FirebaseAnalytics, Firebase.Analytics");
+                if (firebaseAnalyticsType != null) {
+                    if (File.Exists("./analytics_win.dll")) {
+                        string fullSourcePath = Path.GetFullPath("./analytics_win.dll");
+                        // string destinationDirectory = "./"; // This is DestinationDir
+                        string fullDestinationPath = Path.GetFullPath(Path.Combine(DestinationDir, "analytics_win.dll"));
```

Comment:
Nothing ever changes DestinationDir afaik, so this will always be ./analytics_win.dll, which will be the same as the fullSourcePath above. I'm not clear what it is trying to do here.